bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -347,9 +345,8 @@ impl Client {
     }
 
     /// Gets information about the owner of a given access token.
-    pub(crate) fn whoami(&self) -> anyhow::Result<whoami::v3::Response> {
-        RUNTIME
-            .block_on(async move { self.inner.whoami().await.map_err(|e| anyhow!(e.to_string())) })
+    pub(crate) async fn whoami(&self) -> anyhow::Result<whoami::v3::Response> {
+        self.inner.whoami().await.map_err(|e| anyhow!(e.to_string()))
```

Comment:
Preexisting, but isn't it the same as:

```suggestion
        Ok(self.inner.whoami().await?)
```

Reply:
Applied